### DISCUSSION TITLE
Detect the usage of PMF on WPA3 networks and relevant alerting

### DISCUSSION THREAD

**Discussion by cp-cmyk** [nzymedefense/nzyme#769] [Ideas]
It would be interesting if Protected Management Frames (PMF) could be detected by nzyme in case a wireless network uses WPA3-SAE encryption protocol in order to generate relevant alerts in case some unprotected management frames are used without permission in these protected networks by some attacker which is spoofing them towards both directions. I am referring esp. to the deauthentication frames which have such a use case in the wireless world with PMF enabled. It would be great if this can be considered to be something interesting and/or feasible.

**Comment by lennartkoopmann** [comment]
Thank you! Moving this into the 2.x milestone for now.

**Reply by lennartkoopmann** [reply]
Created issue #770
